Applications/Spire/Include/Spire/Toolbar/ToolbarMenu.hpp

```diff
@@ -24,7 +24,7 @@ namespace Spire {
       */
       void add(const QString& text, const QImage& icon);
 
-      void remove_item(int index) override;
+      void remove_item(unsigned int index) override;
```

Comment:
For type `unsigned int`, I think the preferred spelling is `unsigned`. However, this type is rarely used. The most common unsigned type is the pointer size type as it is the correct way to represent array indexes, for example.
Having said that, I have noticed that Qt uses the signed `int` type for most logically unsigned methods. You may want to revert back to `int` or use `std::size_t`, whichever requires fewer casts and enables fewer uses with undefined behavior.

Reply:
Use ```int```.